tests/regression/04-mutex/35-trylock_rc.c

```diff
@@ -49,7 +49,7 @@ void *monitor_thread (void *arg) {
     if (status != EBUSY) {
       if (status != 0)
         err_abort (status, "Trylock mutex");
-      printf ("Counter is %ld\n", counter/SPIN); // RACE!
+      printf ("Counter is %ld\n", counter/SPIN); // RACE
```

Comment:
I don't really understand why this is not an actual race. Maybe worth adding a comment.

Reply:
If `pthread_mutex_trylock` above succeeds to lock, then that access of `counter` is protected by `mutex`. The access in the else branch where locking failed is the racy one. Comparing to 04/36 that's where the difference comes from.